store/tikv/oracle/oracles/pd.go

```diff
@@ -36,7 +36,9 @@ type pdOracle struct {
 	c pd.Client
 	// txn_scope (string) -> lastTSPointer (*uint64)
 	lastTSMap sync.Map
-	quit      chan struct{}
+	// txn_scope (string) -> lastArrivalTSPointer (*uint64)
+	lastArrivalTSMap sync.Map
```

Comment:
`sync.Map` is not necessary, I think use `int64` and `atomic.Store/Load` the timestamp is enough

Reply:
Thanks for your review, I agree with you.